db/migrations/state/0001.sql

```diff
@@ -52,6 +52,18 @@ CREATE TABLE state.forced_batch
     block_num        BIGINT NOT NULL REFERENCES state.block (block_num) ON DELETE CASCADE
 );
 
+CREATE TABLE state.proof
```

Comment:
You should create a new migration file, not modify an existing one.

Reply:
good point @ToniRamirezM, if you agree, I set up a new migration that simply drops and recreates the `state.proof` table with the new fields. This because we probably don't care about existing data on that table if we are migrating to the new aggregator, WDYT? 285299a76d4b8ba14f59cb0eaa92ea764b344b34

Reply:
I agree